documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,287 @@
+
+The migration fails to build the PVC when the destination storage class does not have a configured storage profile. The error logs lack clear information to identify the reason for the failure. link:https://issues.redhat.com/browse/MTV-928[(MTV-928)]
+
+
+.Migration of a VM with NVME disks from vSphere fails
+
+When migrating a virtual machine (VM) with NVME disks from vSphere, the migration process fails, and the Web Console shows that the `Convert image to kubevirt` stage is `running` but did not finish successfully. link:https://issues.redhat.com/browse/MTV-963[(MTV-963)]
+
+
+.Importing image-based VMs can fail
+
+Migrating an image-based VM without the `virtual_size` field can fail on a block mode storage class. link:https://issues.redhat.com/browse/MTV-946[(MTV-946)]
+
+
+.Deleting migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs and data volumes. You must archive a migration plan before deleting it to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+
+.Migrating VMs with independent persistent disk from VMware to OCP-V fail
+
+Migrating Virtual Machines with independent persistent disk from VMware to OCP-V fail. link:https://issues.redhat.com/browse/MTV-993[(MTV-993)]
+
+
+.Migrating VM from one OCP-V cluster to another using timeouts and fails
+
+When trying to migrate VMs from one OCP-V cluster to another using {product-short}, the migration could timeout and fail. link:https://issues.redhat.com/browse/MTV-1052[(MTV-1052)]
+
+
+.Guest operating system from vSphere might be missing
+
+The guest operating system may be missing for VMs from vSphere. vSphere stops reporting the operating system after not receiving updates from VMware tools after some time. link:https://issues.redhat.com/browse/MTV-1046[(MTV-1046)]
+
+
+.{product-short} forklift-controller pod crashes in a loop after configuring vSphere provider
+
+The {product-short} `forklift-controller` pod crashes in a loop, and the product is unusable if vSphere provides disk with null datastore. link:https://issues.redhat.com/browse/MTV-1029[(MTV-1029)]
+
+
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430275[Known Issues] in Jira.
+
+
+
+[id="resolved-issues-26_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+
+.Deleting a provider automatically also deletes the PV
+
+In the previous versions of {product-short}, the PV was not removed when the OVA provider was deleted. This has been resolved in {product-short} 2.6.0, and the PV is automatically deleted when the OVA provider is deleted. link:https://issues.redhat.com/browse/MTV-848[(MTV-848)]
+
+
+.Data loss is no longer possible when migrating VMware VMs with snapshots
```

Comment:
```suggestion
.Fix for data being lost when migrating VMware VMs with snapshots
```
I'm a bit afraid to say "Data loss is no longer possible" - we fixed a bug and we hope it's the only bug that can lead to this